### ISSUE TITLE
vscode reports bad request when attempting to build when build in progress

### ISSUE THREAD

**Issue by tobespc** [eclipse/codewind#3032]
**Codewind version:** latest
**OS:**

**Description:**
Following error message displayed when manually triggering a build 
![Screenshot 2020-05-27 at 12 09 42](https://user-images.githubusercontent.com/13151286/83012490-622a8d00-a013-11ea-8338-e4aac5ba3ee4.png)

**Steps to reproduce:**
Create a project and start it building, then from the overview page click the build button and then the message will appear

**Comment by tetchel** [comment]
That's the response from the portal server, this is a portal bug

**Comment by tobespc** [comment]
retesting now @tetchel , no one else could get this yesterday so it may be just me

**Comment by tobespc** [comment]
recreated it again.  Steps are create a websphere project, start a load run, whilst its in the load running state, click the build button on the project overview page

**Comment by stalleyj** [comment]
So as per Toby's comment above, it's hitting the build api when a load run is in progress that is the issue.  The build API has been modified to reject a build request and return 400 with message when there is a load run in progress (it already did this for a project that is disabled or being deleted).  Note that the API spec is not accurate and we also return a 400 for project not found.  I propose that we update the API to return 404 for project not found and then return 400 "Unable to build project as  a load run is in progress' separately from 'Cannot perform build action ....' msg.  @tetchel Will VSCode (and other IDEs) also need changes ?

**Comment by tetchel** [comment]
No, VS Code does not look for any specific responses or errors from the build API

**Comment by tobespc** [comment]
why can't you display the error message ?

**Comment by tetchel** [comment]
oh i see, the message being sent is the "cannot build because project is in state" one... I should be showing it, I will see why that isn't working. https://github.com/eclipse/codewind/issues/3057

that said, I can't reproduce this for some reason, I always get the 202 response

**Comment by stalleyj** [comment]
@tetchel the change to the API went into master yesterday some point before DCUT

**Comment by tetchel** [comment]
here is the what the current error looks like with the fixed HTTP error reporting

![image](https://user-images.githubusercontent.com/8888438/83276085-7a7ee100-a1a6-11ea-9b28-aa1fb17b48ad.png)
